source/hic_hal/nxp/lpc11u35/gpio.c

```diff
@@ -94,11 +112,19 @@ void gpio_init(void)
     PIN_CDC_LED_IOCON = PIN_CDC_LED_IOCON_INIT;
     LPC_GPIO->SET[PIN_CDC_LED_PORT] = PIN_CDC_LED;
     LPC_GPIO->DIR[PIN_CDC_LED_PORT] |= PIN_CDC_LED;
+#if defined(BOARD_LPC4088DM) || defined(BOARD_LPC4088QSB)
+    // Configure ISPCTRL as output and high
+    // We use two ISP pins to cover different hardware versions
+    LPC_IOCON->TMS_PIO0_12 |= 0x01;
+    LPC_IOCON->PIO1_15 &= ~0x07;
+    gpio_set_isp_pin(1);
+#else
```

Comment:
One option is to use the IO override in source/board/override_*.   I'm not sure if this is flexible enough to meet your needs though.

What is gpio_set_isp_pin used for?  To disable protection on the target, or is it used for something else?